pages/index.js

```diff
@@ -4,5 +4,5 @@ import { useStore } from "../hooks/useStore";
 export default function Home() {
   const articles = useStore((state) => state.articles);
```

Comment:
Why are you using the store here and then in the ArticleListContainer you are using a state to duplicate the state from the store?

Reply:
here is where i display all the search results. At this moment I dont have datenbank yet, so I have to store the results in the global state.

Reply:
oh wait a minute i think u re right. I dont need zustand for keeping the search results here coz I already pass down the results through the props. Sori hehehe